### DISCUSSION TITLE
mtex not loading my ebsd crc file

### DISCUSSION THREAD

**Discussion by j-c-smith** [mtex-toolbox/mtex#1883] [Ask Anything] [answered]
I've attempted to load a crc file on mtex using the import wizard as well using the syntax shown:

>> ebsd = EBSD.load('5_map_data.crc')

I've had no issues with this file type before. I've tried working on the latest mtex release as well as others but still the same response in following error:

Error using check_interfaces
Could not detect file format. You may ask for help in the MTEX forum
(https://groups.google.com/forum/#!forum/mtexmail).

Error in EBSD.load (line 72)
  [interface,options] = check_interfaces(fname{1},'EBSD',varargin{:});

Any help on this would be great!
Jonny

**Comment by zmichels** [comment, marked as answer]
If you don’t already… You’ll need to have both the CRC and CPR file-paor in the same folder. Alternatively you could use a single-file CTF format if you can export or convert to that.ZachOn Jun 26, 2023, at 8:04 PM, j-c-smith ***@***.***> wrote:﻿
I've attempted to load a crc file on mtex using the import wizard as well using the syntax shown:


ebsd = EBSD.load('5_map_data.crc')


I've had no issues with this file type before. I've tried working on the latest mtex release as well as others but still the same response in following error:
Error using check_interfaces
Could not detect file format. You may ask for help in the MTEX forum
(https://groups.google.com/forum/#!forum/mtexmail).
Error in EBSD.load (line 72)
[interface,options] = check_interfaces(fname{1},'EBSD',varargin{:});
Any help on this would be great!
Jonny

**Reply by j-c-smith** [reply]
Thanks, I've now got one of my files running. Only issue is, I need to track down the other CPR files to pair along side the CRC's.

Much appreciated.
